### PR TITLE
Added home-page template with basic styling

### PR DESCRIPTION
**Pull request by MiryalaAkshitha**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,51 +1,13 @@
...
-}
+@import url(https://pro.fontawesome.com/releases/v5.10.0/css/all.css);
+
+:root{
+  --primary-color:var(red);
+  --text-color:var(white);
+  --xs:var(0.625rem);
+  --s:var(1.25rem);
+  --m:var(2rem);
+  --l:var(3.125rem);
```
Comment (sandeepkumardev-zz) [inline src/App.css]: Instead of this, you should write sm, md, xl.

**Review by sandeepkumardev-zz — APPROVED** [review]

File: src/App.css (hunk 1 above)
Comment (MiryalaAkshitha) [reply src/App.css]: sure

File: src/components/Hero/Hero.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import "./Hero.css";
+export const Hero = () => {
+  return (
+    <div className="hero-container">
+      <img
+        className="hero-img"
+        src="https://cdn.shopify.com/s/files/1/0105/1426/8260/files/Best_Women_dresses_Avirate.jpg?v=1580455888"
```
Comment (SyedRashed007) [inline src/components/Hero/Hero.js]: Add an alt tag

**Review by SyedRashed007 — APPROVED** [review]

File: src/components/Hero/Hero.js (hunk 2 above)
Comment (MiryalaAkshitha) [reply src/components/Hero/Hero.js]: sure, I missed it!

**Pull request merged**